rasa/core/channels/twilio.py

```diff
@@ -103,6 +103,14 @@ def from_credentials(cls, credentials: Optional[Dict[Text, Any]]) -> InputChanne
             credentials.get("twilio_number"),
         )
 
+    @classmethod
+    def _is_location_message(cls, request):
+        """Check if the users message is a location."""
+        return (
+            request.form.get("Latitude", None) is not None
+            and request.form.get("Longitude", None) is not None
+        )
```

Comment:
Can you please rephrase it for readability? I would suggest,

```
    @classmethod
    def _is_location_message(cls, request):
        """Check if the users message is a location."""
        return request.form.get("Latitude") and request.form.get("Longitude")
```

Reply:
Done